internal/adapters/operation/operation_storage_redis_test.go

```diff
@@ -912,7 +946,7 @@ func TestCleanOperationsHistory(t *testing.T) {
 			operationsTTLMap := map[Definition]time.Duration{}
 			storage := NewRedisOperationStorage(client, clock, operationsTTLMap)
 
-			operationsReturned, err := storage.ListSchedulerFinishedOperations(context.Background(), schedulerName)
+			operationsReturned, _, err := storage.ListSchedulerFinishedOperations(context.Background(), schedulerName, 0, 10)
```

Comment:
As the test ensures that there is no `operation` the total return should return `zero`. What do you think to `assert.Equal(t, total, 0)`.